src/ios/c_headers/coepicore.h

```diff
@@ -3,3 +3,20 @@
 CFStringRef fetch_new_reports();
 CFStringRef get_reports(int32_t interval_number, int32_t interval_length);
 CFStringRef post_report(const char *to);
+
+// Variant to send finished inputs as JSON. Not continuing this approach for now.
+// CFStringRef submit_symptoms_complete(const char *to);
+
+CFStringRef set_cough_type(const char *to);
```

Comment:
I came across a lib for generating C headers (doubt we have a need for it, but just in case...) : https://github.com/eqrion/cbindgen/

Reply:
I saw it too. But it seems unnecessary for now.